### DISCUSSION TITLE
How to close the process of epmd port

### DISCUSSION THREAD

**Discussion by xhsky** [rabbitmq/rabbitmq-server#2897] [Other]
After I use the command (rabbitmqctl stop) to shut down mq, the epmd process is not shut down. How can I shut down the epmd process while shutting down mq?

**Comment by michaelklishin** [comment]
I will convert this issue to a GitHub discussion. Currently GitHub will [automatically close and lock the issue](https://github.com/github/feedback/discussions/3197) even though your question will be transferred and responded to elsewhere. This is to let you know that we do not intend to ignore this but this is how the current GitHub conversion mechanism makes it seem for the users :(

**Comment by michaelklishin** [comment]
``` shell
epmd -kill
```

**Comment by michaelklishin** [comment]
Note that `epmd` has [access restrictions](https://erlang.org/doc/man/epmd.html) that make it perfectly fine to leave running in most cases.
